extensions/slack/src/search.tsx

```diff
@@ -1,82 +1,95 @@
-import { ActionPanel, Icon, Image, List } from "@raycast/api";
+import { ActionPanel, Action, Icon, Image, List } from "@raycast/api";
 
-import { CacheProvider, onApiError, useChannels, useGroups, useUsers } from "./shared/client";
+import { User, onApiError, useChannels, useGroups, useUsers } from "./shared/client";
 import { UpdatesModal } from "./shared/UpdatesModal";
+import { withSlackClient } from "./shared/withSlackClient";
+import { useFrecencySorting } from "@raycast/utils";
 import { OpenChannelInSlack, OpenChatInSlack, useSlackApp } from "./shared/OpenInSlack";
 
-export default function Command() {
+function Search() {
   return (
-    <CacheProvider>
-      <UpdatesModal>
-        <SlackList />
-      </UpdatesModal>
-    </CacheProvider>
+    <UpdatesModal>
```

Comment:
Probably not. This modal is a bit obscure to me by the way. Why has it been added? Wouldn't it be better to replace that by a failure toast if a Slack API Call fails and ask to change the scopes there?

Reply:
Yeah it was before my time too, and my best guess is that certain new commands required permission changes. A failure toast makes more sense for permission failures. However, I don't want to cause scope creep and we can handle this removal as part of a separate issue/PR thread.

Reply:
I think we'll have to remove as part of this PR though because people logging in with OAuth will see the update modal pop up. I'm trying to see how we can provide a better error experience.

Reply:
For sure, then we can remove this in favor of toasts in this PR. Or at the very least only render it if `xoxp-` token is being set.

Reply:
All right, it's been done!